util/CMakeLists.txt

```diff
@@ -1,5 +1,14 @@
-find_package(Boost 1.54 REQUIRED)
+find_package(Boost 1.54 REQUIRED system thread chrono date_time)
```

Comment:
For static libraries, this order needs to be: `x depends_on y depends_on z`. So this should probably be `date_time chrono system`. I think `thread` is no longer needed.